NEWS.md

```diff
@@ -11,9 +11,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * `Distributions.jl`, `RecursiveArrayTools.jl` and `HybridArrays.jl` were moved to weak dependencies to reduce load time and improve extensibility.
 * `translate_diff`, `inv_diff` and thus `apply_diff_group`, are available for all the groups with invariant tangent vector storage
-* `adjoint_action` takes a direction argument
-* `adjoint_action!` is the necessary method to implement in any given group for the above to work properly
-* On groups with invariant vector storage, `exp` and `log` are based on `exp_lie` and `log_lie` and thus group invariant
+* `adjoint_action` takes a direction argument; by default it is `LeftAction`.
+* `adjoint_action!` is the necessary method to implement in groups with left-invariant tangent vector storage.
```

Comment:
```suggestion
* `adjoint_action!` is the necessary method to implement in groups with left-invariant tangent vector representation.
```

Am I right here? I would make the distinction as follows:
* __representation__ is the question/indication how the vector is represented mathematically (in the Lie algebra, in a tangent space, ....)
* __storage__ is the type of data stored, a single array or a sparse array

The abstract type below also is called representation.

Reply:
Yes, representation is more suitable here.

Reply:
I will try to find time – maybe on the weekend – to check that these two terms are consistently used throughout. I think there was a mixup on the other PR as well somewhen.

Reply:
Yes, that's a good idea, thanks! The PR is now more or less complete, I'm only polishing it and consider if I should maybe look for anything else to break.